### ISSUE TITLE
scheduler: test-performance.sh compile error.

### ISSUE THREAD

**Issue by jayunit100** [kubernetes/kubernetes#32223]
_Problem_

Looks like the test-performance.sh utility doesn't work because of a compile change in test/integration/framework `DeleteAllEtcdKeys` ?

_Solution_ 

Reimplement DeleteAllEtcdKeys ? 

_Details_

```
➜  perf git:(master) ✗ ./test-performance.sh 
~/Development/kubernetes/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/component/scheduler/perf ~/Development/kubernetes/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/component/scheduler/perf
etcd --advertise-client-urls http://127.0.0.1:2379 --data-dir /var/folders/nc/49qzdyt931bg1hgpc2bz277r0000gn/T/test-etcd.XXXXXX.bOkDF0Nl --listen-client-urls http://127.0.0.1:2379 --debug > "/dev/null" 2>/dev/null
Waiting for etcd to come up.
+++ [0907 15:57:58] On try 2, etcd: : http://127.0.0.1:2379
{"action":"set","node":{"key":"/_test","value":"","modifiedIndex":4,"createdIndex":4}}
+++ [0907 15:57:58] performance test start
```

then a compile error appears...

```
# k8s.io/kubernetes/test/component/scheduler/perf
./util.go:47: undefined: "k8s.io/kubernetes/test/integration/framework".DeleteAllEtcdKeys
FAIL    k8s.io/kubernetes/test/component/scheduler/perf [build failed]
```

**Comment by jayunit100** [comment]
cc @kubernetes/sig-scheduling

**Comment by jayunit100** [comment]
whoops, dupe of #30812

**Comment by timothysc** [comment]
closed by #32520
